src/SmartComponents/Advisor/Advisor.cy.js

```diff
@@ -0,0 +1,59 @@
+import React, { Suspense } from 'react';
+import Advisor from './Advisor';
+import Loading from '../../PresentationalComponents/Loading/Loading';
+import { cvesInterceptors, rulesInterceptors, systemsInterceptors } from '../../../cypress/support/interceptors';
+import Masonry from 'react-masonry-css';
+import { Grid } from '@patternfly/react-core/dist/esm/layouts';
+
+before(() => {
+    cy.mockWindowChrome();
+});
+
+const breakpointColumnsObj = {
+    default: 2,
+    992: 1
+};
+
+const defaultValues = {
+    categoryTypes: [
+        { name: 'Availability', value: '62' },
+        { name: 'Stability', value: '10' },
+        { name: 'Performance', value: '20' },
+        { name: 'Security', value: '35' }]
+};
+
+const AdvisorCard = () => {
+    return (
+        <Grid hasGutter>
+            <Masonry
```

Comment:
What is this for?

Reply:
I am replicating the component structure in the Dashboard.js
![image](https://github.com/RedHatInsights/insights-dashboard/assets/62722417/063a1ae1-ae16-443a-87dc-def1ee4edc21)

Reply:
It might be better to just wrap the `VulnerabilityCard` only in what it really needs, likely just the `Router` and maybe a store provider.

Introducing more than needed, means more to investigate if something fails.

Reply:
We can remove Grid component but we have to keep Masonry, there are at least 2 reasons for that.
1) If it breaks on the Grid or Masonry level we will know about that
2) It looks different from the actual representation without the Masonry component. 
This is how it looks with Masonry
![image](https://github.com/RedHatInsights/insights-dashboard/assets/62722417/68f39b77-2d95-4edd-a403-c675a730eb28)

And this is without it
![image](https://github.com/RedHatInsights/insights-dashboard/assets/62722417/3e62a0f4-425d-4db8-ac65-915391b76527)

Reply:
@Fewwy Other than the width, I see no visual difference. and it just being wider is very acceptable.

Reply:
Ok :) I will remove both Grid and Masonry components then

Reply:
@bastilian I removed the masonry and grid from this and 2 other files